internal/pkg/cli/storage_init.go

```diff
@@ -730,18 +747,23 @@ func (o *initStorageOpts) Execute() error {
 	}
 	for _, addon := range addonBlobs {
 		path, err := o.ws.Write(addon.blob, addon.path)
-		if err != nil {
-			e, ok := err.(*workspace.ErrFileExists)
-			if !ok {
-				return err
-			}
-			return fmt.Errorf("addon file already exists: %w", e)
+		if err == nil {
+			log.Successf("Wrote CloudFormation %s at %s\n",
+				addon.description,
+				color.HighlightResource(displayPath(path)),
+			)
+			continue
 		}
-		path = displayPath(path)
-		log.Successf("Wrote CloudFormation %s at %s\n",
+		_, ok := err.(*workspace.ErrFileExists)
+		if !ok {
```

Comment:
nit: I think it might be safer for us to use the `errors.As` function in case the workspace methods ever wrap this error.

```go
var ferr *workspace.ErrFileExists
if !errors.As(err, ferr) { return err }
```